src/assets/css/decred-v5.css

```diff
@@ -4165,7 +4161,7 @@ html[dir="rtl"] .lang-menu {
 }
 
 .left-wrapper {
-  width: 320px;
```

Comment:
This is a site wide change - I don't think this is correct, please reference the GH issue